### PR TITLE
add packages to Imports: in DESCRIPTION

### PR DESCRIPTION
**Pull request by jimrothstein**
This should be cleaner.

Doing some reading - thanks.

jim

### CODE REVIEW

**Comment by JosiahParry** [comment]
Looks great! 

Can you do a few more things?

1. Bump version to `0.0.9001` in the `DESCRIPTION` file?
2. Add yourself as a contributor to the package `role = "ctb"` in the DESCRIPTION file?
3. Create a NEWS.md and outline the changes you've made? See http://r-pkgs.had.co.nz/release.html#news

I'll merge after that!

**Comment by jimrothstein** [comment]
Will do ...by Monday.    So far the R portion are minimal;  it is the GIT portion that ties me in knots.  Have all the tools, just need to practice.

**Comment by jimrothstein** [comment]
Please look over my weekend work on PR.   Thx.


NOTE:  
Tried to use/test each of package's R functions  (using separate code;  not in package, not in PR).
Found a few errors,  probably can fix by reviewing API documentation.   Might be great time to do this!
Could you review Issue #5 ?

File: R/utils.R [hunk 1]
```diff
@@ -7,7 +7,7 @@ set_os_key <- function(api_key) {
 }
 
 
-
+#' @export
```
Comment (JosiahParry) [inline R/utils.R:19]: This should still be an internal function. Do you mind changing this to `#' @keywords internal`?

File: NEWS.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+-V or --variable
+--pdf-engine=xelatex
+
+PANDOC EXAMPLES:
+https://learnbyexample.github.io/tutorial/ebook-generation/customizing-pandoc/
+
+MARKDOWN GUIDE:
+https://www.markdownguide.org/basic-syntax/
+
+-->
```
Comment (JosiahParry) [inline NEWS.md]: Do you mind removing this comment?

File: NEWS.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+***
+
+### 0.0.9001  Misc Fixes
+
+*	add imports to DESCRIPTION (#3).
+*	add httr::stop_for_status() to GET() (#4).
+* begin NEWS.md (#7, @jimrothstein).
+
+<!--
```
Comment (JosiahParry) [inline NEWS.md]: And this one? Thanks!

File: R/utils.R (hunk 1 above)
Comment (jimrothstein) [reply R/utils.R:19]: I use set_os_key and get_os_key in my non-package code,  ie as a user.  I do not see the functions if remove @export.
Do I misunderstand?
Also,  [r-pkgs version 2](https://r-pkgs.org/namespace.html) :  
"You export functions that you want other people to use."

File: NEWS.md (hunk 2 above)
Comment (jimrothstein) [reply NEWS.md]: Comments:    Certainly can remove.   

I use neovim (not RStudio) and bottom line are control codes for page formatting.   Need to figure out work around.

**Comment by jimrothstein** [comment]
I need to remove last commit message "Merge branch 'jim......."    Fear making it worse.

File: R/utils.R (hunk 1 above)
Comment (JosiahParry) [reply R/utils.R:19]: If it is to be exported, please document the function. I personally do not see the need but another exported function cannot hurt. 

Please change to #' @keywords internal or add additional documentation. Thanks!

File: NEWS.md (hunk 2 above)
Comment (JosiahParry) [reply NEWS.md]: Given that this is just a comment I think it's fine to leave for the time being.

Comment (jimrothstein) [reply NEWS.md]: It is still there??   I am trying to see what you see on your end.

**Pull request merged**